modules/imgproc/src/undistort.cpp

```diff
@@ -40,8 +40,10 @@
 //
 //M*/
 
+
 #include "precomp.hpp"
 #include "opencv2/imgproc/detail/distortion_model.hpp"
+#include <cmath>
```

Comment:
can we move `#include <cmath>` into `imgproc/src/precomp.hpp`? (and probably do the same with other modules, so that we have less such includes)